### ISSUE TITLE
Automatically download submodules of git sources

### ISSUE THREAD

**Issue by kylef** [CocoaPods/cocoapods-try#38]
To allow you to store example projects in separate repositories. It would be nice if they can be stored in a submodule.

Example here: https://github.com/the-hypermedia-project/Hyperdrive/tree/kylef/example

**Comment by kylef** [comment]
This might make more sense to provide it in a config file `.cocoapods-try.yaml` in the root of a repository. This could allow other features:

```yaml
pre: pod install
project:
  - Example/Polls.xcworkspace
submodules: true
```

**Comment by orta** [comment]
To expand further, what if this was a total `.cocoapods.yml` file, which included all [CocoaDocs settings too](https://github.com/orta/ARAnalytics/blob/master/.cocoadocs.yml) as well as all this stuff

**Comment by neonichu** [comment]
Implemented by #39
